go.work

```diff
@@ -1,4 +1,6 @@
-go 1.21
+go 1.22
+
+toolchain go1.22.5
```

Comment:
IDK is it a good idea to force a specific toolchain?!?

Reply:
Indeed, it's tricky usually. But it's default behavior of `go mod tidy` to add it, whenever go directive is bumped.